dist/src/main/java/io/camunda/zeebe/shared/MainSupport.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.shared;
+
+import io.camunda.zeebe.broker.Loggers;
+import io.camunda.zeebe.util.error.FatalErrorHandler;
+import org.springframework.boot.WebApplicationType;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+
+/**
+ * Utility class for shared behavior in application `main`, i.e. program entry points. This helps
+ * ensure all executables have some sane defaults (e.g. applying system properties, per thread
+ * uncaught exception handler, etc.)
+ */
+public final class MainSupport {
```

Comment:
:thought_balloon: I extracted some common `Main` stuff here. I'm not super keen on the name or even how it's done, but I prefer this to an abstract base class :shrug: